docs/reference/semaphore-changelog.md

```diff
@@ -10,6 +10,18 @@ This page is updated on a weekly basis.
 
 ### Week of November 2, 2020
 
+**(Improved) Ubuntu 18.04 image update**  
+
+**(Updated)** Language versions:
+
+- Aws-cli 1.18.159 -> 1.19.8
+- Azure-cli 2.13.0 -> 2.14.0
+- Git 2.28 -> 2.29
+- Heroku 7.46.0 -> 7.47.0
+- OpenJDK 8u265 -> 8u272
+- Openjdk 11.0.8 -> 11.0.9
+- Php 7.3.23 -> 7.3.24
```

Comment:
A small style thing but I see that there are both `PHP` and `Php` notations throughout the changelog. [The previous one is PHP](https://docs.semaphoreci.com/reference/semaphore-changelog/#week-of-october-5-2020). Maybe we can stick with just one of them, but there might be a reason behind this I'm missing.

Reply:
Good POINT !!! changed to PHP